tests/integration/test_sqlalchemy_integration.py

```diff
@@ -89,13 +91,15 @@ def test_define_and_create_table(trino_connection):
         sqla.Table('users',
                    metadata,
                    sqla.Column('id', sqla.Integer),
+                   sqla.Column('guid', UUID),
```

Comment:
I think this should be sqla.Uuid instead. See https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.Uuid

Reply:
Uuid only works in v2 of sqlalchemy. So you will need to ensure it only works in that version.